File: 2270-number-of-ways-to-split-array/2270-number-of-ways-to-split-array.py
class Solution:
    def waysToSplitArray(self, nums: List[int]) -> int:
        left, right = nums[0], sum(nums[1:])
        count = 0

        if left >= right: count += 1
        for i in range(1, len(nums)-1):
            left += nums[i]
            right -= nums[i]
            if left >= right: count += 1

        return count

File: 2270-number-of-ways-to-split-array/2270-number-of-ways-to-split-array.cs
public class Solution {
    public int WaysToSplitArray(int[] nums) {
        long left = nums[0], right = nums[1];
        int n = nums.Length, count = 0;
        for(var i=2; i<n; i++) right += nums[i];

        if(left >= right) count++;
        for(var i=1; i<n-1; i++){
            left += nums[i];
            right -= nums[i];
            if(left >= right) count++;
        }

        return count;
    }
}